inc/audiences/rest_api/namespace.php

```diff
@@ -0,0 +1,197 @@
+<?php
+/**
+ * Audience REST API.
+ *
+ * @package altis-analytics
+ */
+
+namespace Altis\Analytics\Audiences\REST_API;
+
+use const Altis\Analytics\Audiences\POST_TYPE;
+use function Altis\Analytics\Audiences\get_estimate;
+use WP_Error;
+use WP_Post;
+use WP_REST_Request;
+use WP_REST_Response;
+use WP_REST_Server;
+
+function setup() {
+	add_action( 'rest_api_init', __NAMESPACE__ . '\\init' );
+}
+
+/**
+ * Register audience API endpoints.
+ */
+function init() {
+	// Fetch data for available fields and possible values.
+	register_rest_route( 'analytics/v1', 'audiences/fields', [
+		[
+			'methods' => WP_REST_Server::READABLE,
+			'callback' => 'Altis\\Analytics\\Audiences\\get_field_data',
+			'permissions_callback' => __NAMESPACE__ . '\\permissions',
+		],
+		'schema' => [
+			'type' => 'array',
+			'items' => [
+				'type' => 'object',
+				'properties' => [
+					'name' => [
+						'type' => 'string',
+						'required' => true,
+					],
+					'label' => [
+						'type' => 'string',
+						'required' => true,
+					],
+					'type' => [
+						'type' => 'string',
+						'required' => true,
+					],
+					'data' => [
+						'type' => 'array',
+						'items' => [
+							'type' => 'object',
+							'properties' => [
+								'value' => [ 'type' => 'string' ],
+								'count' => [ 'type' => 'number' ],
+							],
+						],
+					],
+					'stats' => [
+						'type' => 'object',
+						'properties' => [
+							'count' => [ 'type' => 'number' ],
+							'min' => [ 'type' => 'number' ],
+							'max' => [ 'type' => 'number' ],
+							'avg' => [ 'type' => 'number' ],
+							'sum' => [ 'type' => 'number' ],
+						],
+					],
+				],
+			],
+		],
+	] );
+
+	// Fetch an audience size estimate.
+	register_rest_route( 'analytics/v1', 'audiences/estimate', [
+		[
+			'methods' => WP_REST_Server::READABLE,
+			'callback' => function ( WP_REST_Request $request ) : WP_REST_Response {
```

Comment:
These callbacks (this and the validate/sanitize below) should be split into proper functions to help with readability of the file and route definition.